code/src/component/QuestionOne.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import 'component/questionone.css'
+import 'component/fonts.css'
```

Comment:
You are already inside the component map
```suggestion
import 'questionone.css'
import 'fonts.css'
```